### DISCUSSION TITLE
چه گروه‌های سیاسی بین مخالفین جمهوری اسلامی وجود داره و در چه موضوعاتی اختلاف دارن؟

### DISCUSSION THREAD

**Discussion by unlikely-alice** [WeAreMahsaAmini/Core#499] [Q&A]
دقیقا چه گروه‌های سیاسی بین مخالفین وجود داره و اختلافشون سر چه مسائلی هست که باهم متحد نمی‌شن؟

برای پیروزی باید متحد شد. من می‌خوام بدونم دلایل به وجود نیومدن این اتحاد چی هستش. احتمالا دلایل مختلفی داره ولی بخشی از اون دلایل اختلافات بین مخالفین هست. سوالم فقط در مورد اون اختلافات هست. باید اختلافات رو خوب شناسایی و دسته‌بندی کنیم، ببینیم کدوم ها رو می‌تونیم بعد از انقلاب حل کنیم و کدوم‌ها لازمه‌ی اتحاد برای انقلاب هستند.

**Comment by arandomgstring** [comment]
We have monarchists, republicans, and even anarchists, etc, each with their own ideology and important figures; yet none of these figures has ever published a clear proposal for the constitution after revolution (as far as I am aware)! Why? I don't know. But I do know that as long as we have no constitution from these groups, we cannot compare them logically at all. So in the end, even if revolution does happen (which is unlikely, given the diversity of people right now), a better known party with not necessary best ideology about democracy will rule over people, resulting in another tragedy in the next few years. 

I can only wish these well-known characters write a constitution with their own ideology and publish it for people. So that people actually "choose" among these ideologies and fight for it to the end. Even Iran's authoritarian regime can say vague words such as democracy, human rights, good economy, etc everyday. Everyone can say these words. But what's the basis and constitution for it? No where. The way I see it, even revolutionist themselves haven't taken this revolution seriously yet, otherwise, after all these years we would've had at least a few proposed detailed constitution, which could've become the very basis of unification in the first place. When something like that doesn't exist, on what basis people can become unified? This is why we have this great diversity. Islamic revolution didn't need to deal with this problem too much, because the basis was Islam, and a specific group, "Akhoond" were in control of it (clear enough, no?). Other groups existed too, but the most notable figure with highest power, Khomeini, belonged to this particular group. What about now? Everyone yell democracy, woman rights, etc, without basis. If you don't know what you are fighting for, will you ever seriously fight?
